backend/functions/package.json

```diff
@@ -10,7 +10,7 @@
     "logs": "firebase functions:log"
   },
   "engines": {
-    "node": "16"
+    "node": "12"
```

Comment:
Uhhh you may have to change this back or change it to 10 because that's what I just did or you can just fix changes during merge.